collection-scripts/gather

```diff
@@ -28,3 +29,31 @@ resources+=(clusterautoscaler machineautoscaler)
 for resource in ${resources[@]}; do
     /usr/bin/openshift-must-gather inspect ${resource}
 done
+
+# Collect System Audit Logs
+AUDIT_LOG_PATH="${BASE_COLLECTION_PATH}/audit_logs/"
+mkdir -p ${AUDIT_LOG_PATH}
+/usr/bin/oc adm node-logs --role=master --path=openshift-apiserver/ > ${AUDIT_LOG_PATH}/openshift-apiserver_audit.log
+/usr/bin/oc adm node-logs --role=master --path=kube-apiserver/ > ${AUDIT_LOG_PATH}/kube-apiserver_audit.log
+
+# Collect System Service Logs
+SERVICE_LOG_PATH="${BASE_COLLECTION_PATH}/host_service_logs/"
+MASTER_SERVICE_LOG_PATH="${SERVICE_LOG_PATH}/masters/"
+NODE_SERVICE_LOG_PATH="${SERVICE_LOG_PATH}/nodes/"
+
+# Service Lists
+GENERAL_SERVICES=(kubelet crio)
+MASTER_SERVICES+=${GENERAL_SERVICES[@]}
+NODE_SERVICES+=${GENERAL_SERVICES[@]}
+
+# Collect Master Service Logs
+mkdir -p ${MASTER_SERVICE_LOG_PATH}
+for service in ${MASTER_SERVICES[@]}; do
+    /usr/bin/oc adm node-logs --role=master -u $service > ${MASTER_SERVICE_LOG_PATH}/${service}_service.log
```

Comment:
When I see this, i think of the `queue` function in CI that will run in parallel and wait.  @sanchezl can you link him.

Reply:
Just fork (end the `oc` command with `&` ) and add a call to `wait` after the `done` below.

If you (@deads2k) want the process limiting capabilities of `queue`, then I can provide some sample code for that instead.

Reply:
My updated PR has this queuing in it.